internal/gatewayapi/testdata/backendtrafficpolicy-strategic-merge.out.yaml

```diff
@@ -0,0 +1,278 @@
+backendTrafficPolicies:
+- apiVersion: gateway.envoyproxy.io/v1alpha1
+  kind: BackendTrafficPolicy
+  metadata:
+    creationTimestamp: null
+    name: policy-for-route
+    namespace: default
+  spec:
+    connection:
+      bufferLimit: 100M
+    mergeType: StrategicMerge
+    targetRef:
+      group: gateway.networking.k8s.io
+      kind: HTTPRoute
+      name: httproute-1
+  status:
+    ancestors:
+    - ancestorRef:
+        group: gateway.networking.k8s.io
+        kind: Gateway
+        name: gateway-1
+        namespace: envoy-gateway
+        sectionName: http
+      conditions:
+      - lastTransitionTime: null
+        message: Merged with policy envoy-gateway/policy-for-gateway
+        reason: Overridden
```

Comment:
Should we use `Merged` instead of `Overridden` ?

Reply:
I have no strong opinion.

Reply:
I think `Merged` is a better fit here, as `Overriden` doesn't accurately describe the behavior.

Reply:
+1 to Merged, and we can improve this later in v1.5 as part of https://github.com/envoyproxy/gateway/issues/5745

Reply:
https://github.com/envoyproxy/gateway/pull/5795/commits/da4ee6cc598e97bfba09a89a87c68c37780978e1